### PR TITLE
util/printer: add race enabled in TiDBInfo

### PR DESCRIPTION
**Pull request by coocood**
Makes it easy to check if the binary is build with race.

### CODE REVIEW

**Comment by coocood** [comment]
@shenli @tiancaiamao @zimulala PTAL

File: util/israce/norace.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !race
+
+package israce
+
+// RaceEnabled checks if race is enabled.
+const RaceEnabled = false
```
Comment (shenli) [inline util/israce/norace.go]: I am confused about this PR.
Is the code in line 13 make it work?

Comment (coocood) [reply util/israce/norace.go]: Yes

**Comment by shenli** [comment]
LGTM
Learn a lot!

**Comment by shenli** [comment]
@tiancaiamao PTAL

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**